src/Service/IgnitionErrorListener.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Spatie\SymfonyIgnitionBundle\Service;
+
+use ob_start, ob_get_contents, ob_end_clean;
```

Comment:
This is just a personal preference, but I think each import statement should be in its own line.
```suggestion
use function ob_get_clean;
use function ob_start;
```

Reply:
Agreed & accepted.